K2Bridge/ElasticQueryTranslator.cs

```diff
@@ -104,6 +104,13 @@ public QueryData TranslateQuery(string header, string query)
                 docValueFields = new List<string>();
                 elasticSearchDsl.DocValueFields?.ForEach(item => docValueFields.Add(item.Field));
 
+                // We currently only support a single primary aggregation, so we take the first element from the dictionary and retrieve its type name
+                string primaryAggregation = null;
+                if (elasticSearchDsl.Aggregations != null && elasticSearchDsl.Aggregations.Count > 0)
+                {
+                    primaryAggregation = elasticSearchDsl.Aggregations.First().Value.PrimaryAggregation.GetType().ToString().Split('.')[^1];
```

Comment:
Does obj.GetType().Name can be used instead of obj.ToString().Split('.')[^1]?

Reply:
Good catch, thanks!